samples/sample-app-storage/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: "com.android.application"
 
 android {
     compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    buildToolsVersion "29.0.3"
```

Comment:
Is there any reason we are hard-coding this instead of using buildToolsVersion "$androidBuildToolsVersion" like in our root build.gradle file?

Reply:
I went back and forth on this, but since this is a sample, I assume we would want a customer to be able to download the project and open it up and have it just work. It's a maintenance burden to keep these dependency versions up to date, but I think it would be more confusing to the customer if they were just all referencing variables defined elsewhere.

Actually given that, we should probably use a local copy of the lint.xml file too. I'll fix that shortly.